src/lib/Scenes/Show/Components/Artists/index.tsx

```diff
@@ -1,14 +0,0 @@
-import React from "react"
```

Comment:
I renamed this as we run into issues with Relay query fragment naming in the tests. For `Artists/index.tsx`, Danger expects a new test is added for `index-tests.tsx`, which creates issues if we want to start using `MockRelayRenderer` in these tests.